wallet/src/account/mod.rs

```diff
@@ -529,7 +535,7 @@ impl Account {
         median_time: BlockTimestamp,
     ) -> WalletResult<BTreeMap<Currency, Amount>> {
         let amounts_by_currency = group_utxos_for_input(
-            self.get_utxos(utxo_types, median_time).into_iter(),
+            self.get_utxos(utxo_types, median_time, false).into_iter(),
```

Comment:
It should be possible to get the balance, confirmed and not confirmed. The GUI may need to see which is which (the CLI as well). Knowing whether the balance is confirmed or not is important.

Reply:
added an argument for controlling this